README.md

```diff
@@ -13,6 +13,7 @@ Codemod to convert React PropTypes to TypeScript types.
 - Supports files with multiple components
 - Copies JSDoc comments to the generated TypeScript types
 - Option to remove or preserve PropTypes after converting to TS
+- Choose between interface and type alias
```

Comment:
Not a important feature for the highlights section.

```suggestion
```

Reply:
Addressed